styles.css

```diff
@@ -0,0 +1,140 @@
+@import url("https://rsms.me/inter/inter.css");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Inter", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+}
+
+/* navBar */
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  font-weight: 500;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+li {
+  padding: 1%;
+  color: #172b4d;
+}
+
+a {
+  text-decoration: none;
+  color: #172b4d;
+}
+
+/* the whole content */
+
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-image: url(./utils/Header-llustration-desktop@2x.png);
+  height: 100vh;
+  width: 100%;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 1970px 1100px;
+  background-attachment: fixed;
+  background-position-y: top;
+}
+
+.welcome-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: 30px;
+  padding-left: 10%;
+  max-width: 90%;
+  padding-right: 10%;
+  margin-top: 15%;
+}
+
+.welcome-section > * {
+  margin-top: 15px;
+}
+
+.title {
+  font-size: 60px;
+  color: #172b4d;
+}
+
+.subTitle {
+  color: #36b37e;
+}
+
+.bio {
+  color: #172b4d;
+  font-weight: 200;
+  max-width: 70%;
+  text-align: center;
+}
+
+.socials {
+  display: flex;
+  margin-top: 50px;
+}
+
+.fa {
+  justify-content: space-between;
+  margin-right: 15px;
+  margin-left: 15px;
+}
+
+.fa:hover {
+  opacity: 0.8;
+  color: #36b37e;
+}
+
+@media (max-width: 710px) {
+  .nav-item {
+    display: none;
+  }
+
+  .fa-bars {
+    display: inline-block;
+    font-size: 50px;
+  }
+
+  ul {
+    justify-content: flex-end;
+  }
+
+  .title {
+    margin-top: 80px;
+    font-size: 50px;
+    color: #172b4d;
+  }
+
+  .subTitle {
+    color: #36b37e;
+    font-size: 25px;
+    text-align: start;
+  }
+
+  .bio {
+    text-align: start;
+    max-width: 100%;
+  }
+}
```

Comment:
Kindly ensure that the font size and color of the main title and bio paragraph match that given in the design. Check the above comment for more on why you should stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185386137-6a040f10-f7f7-437a-ab86-83b60d6032eb.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/185386255-473ce9cf-004f-4dbe-a4d1-dfff5762f5ba.png)